types/recharts/index.d.ts

```diff
@@ -491,11 +492,11 @@ export interface RadialBarChartProps {
 }
 export class RadialBarChart extends React.Component<RadialBarChartProps, {}> {}
 
-export interface RectangleProps {
+export interface RectangleProps extends Partial<CSSStyleDeclaration> {
 	x?: number;
 	y?: number;
-	width?: number;
-	height?: number;
+	// width?: number;
+	// height?: number;
```

Comment:
If the property is not there any more, could you please *remove* it instead of commenting it out?